### ISSUE TITLE
Install @babel/eslint-parser

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/WC-Generator#266]
## Describe the bug

When migrating repos, there are issues with bringing some code forward and our new eslint rules. This issue is specifically seen when all other `@babel` resources are removed as part of the deprecation of IE support as they are not needed to generate a legacy ES5 version of a bundled component. 

## To Reproduce

1. Take any legacy repo
1. Migrate to latest generator build
1. Run build
1. See error

## Expected behavior

We need to allow for this legacy code to carry forward and still apply eslint rules. 

## Solution

Install @babel/eslint-parser

This issue was recently discovered with work done to auro-tokenlist and was recently seen again with a migration of auro-hyperlink.

**Comment by geoffrich** [comment]
The underlying issue in auro-tokenlist seemed to be using the newer null coalescing syntax, e.g. `index.wcag?.ratio`. I'd rather not add a babel dependency if we can avoid it. Can we [update our ecmaVersion](https://stackoverflow.com/a/63239666/14808988) instead?

Updating the auro-tokenlist eslintrc to the following fixed the lint issue without installing `@babel/eslint-parser` as a dependency.

```js
{
  "extends": ["@aurodesignsystem/eslint-config"],
  "parserOptions": {
    "ecmaVersion": "2020"
  }
}
```

**Comment by blackfalcon** [comment]
Well... this is getting interesting. It turns out that this addition is not the fix after all. I also just discovered that linters have been silently failing for some time now as the jsdoc linter isn't installed by default. https://github.com/AlaskaAirlines/WC-Generator/pull/265/commits/e42b62ff0d26467f45a7feef86f1f8bfc7022f8c

Closing this issue. The real issue now is 

```
When linting something other than CSS, you should install an appropriate syntax, e.g. "postcss-scss", and use the "customSyntax" optio
```
